frontend/src/App.vue

```diff
@@ -72,13 +72,6 @@ export default {
 @import 'src/scss/global';
 @import '~@mdi/font/css/materialdesignicons.css';
 
-.v-btn.ec-drawer-open,
```

Comment:
Why was this in the App.vue?

Reply:
Styling has evolved over time, sometimes not as clean as I wished.